### DISCUSSION TITLE
只能卸载重装用一次，之后下线再连就和我本地网冲突

### DISCUSSION THREAD

**Discussion by cheerfulzh** [getlantern/lantern-forum-cn#186] [Q&A]
2019年开始用蓝灯，平时遇到外界原因不可以用也就算了。可是大约从春节之前，一直登录困难。卸载重装之后，可以使用，但只能用一次，下线之后再连就不行了，而且本地网络会跟着断，要重置代理服务器才可以连上网络。系统是win10，防火墙已关闭。请教有没有遇到类似问题以及解决办法，感谢！

**Comment by MaxMKMax** [comment]
您好，谢谢您的反馈。蓝灯已经在正视这个问题了。 我们正在修复问题和进行优化。请您耐心等待。非常抱歉影响了您的体验感。谢谢您的等待与支持。

您可以帮助我们在应用程序中提交报告您的问题吗？您可以访问Lantern应用程序并提交log/ 报告。

1. go to Support (转到支持)
2. select Report an Issue (选择报告问题)
3. choose issue type (选择问题类型)
4. include additional details (包括额外的详细信息)
5. hit Send Report (点击发送报告)

当您在app提交问题报告后您会收到邮件. 邮件标题上有 8 位数号码. 请回来这里留言您的log/ticket号码. 因为我需要拿您的反馈报告号码来和科技团队沟通您遇到的问题. 不然我无法提交您遇到的问题给科技团队.

有了这些资料，我们科技团队便可以尽快解决问题。您可以在留言区给我知道。谢谢。
我们将非常感谢您。

**Comment by cheerfulzh** [comment]
报告发送不出去，发邮件给support@getlantern.org申请退款，直接退信。行吧，不会再用了。
